buildflow/core/options/runtime_options.py

```diff
@@ -49,6 +49,7 @@ class RuntimeOptions(Options):
     # misc
     log_level: str
     checkin_frequency_loop_secs: int = 5
+    autoscale_frequency_secs: int = 60
```

Comment:
can see this making sense in runtime since runtime actor will use it, but maybe this would be a better fit in autoscaler options?

Reply:
I could see us making the autoscaler a separate actor from runtime down the road, so might help keep that options open

Reply:
moved to autoscaler options